packages/tokens/src/shadows.ts

```diff
@@ -0,0 +1,10 @@
+import { transparentize } from 'polished';
+import { palette } from '@leafygreen-ui/palette';
+
+const shadows = {
```

Comment:
I'm actually not sure I agree with this approach. What does defining shadows in tokens help us with if they aren't intended to be reused across components?